ada/ast.py

```diff
@@ -2352,8 +2352,11 @@ def base_subp_declarations():
         NOTE: for the moment this only works for tagged types. Remains to be
         seen if we need to extend it.
         """
+        # We don't want to take into account Entity's metadata, as the result
+        # of this property shouldn't depend upon how this node was retrieved.
+        entity_no_md = Var(Entity.without_md.cast(BasicDecl))
```

Comment:
Not sure why you have a var since you only use this once inside a then, seems like it would be better inlined ?

Reply:
done!